Tests/MarathonTests/MarathonTests.swift

```diff
@@ -467,6 +467,92 @@ class MarathonTests: XCTestCase {
                throwsError: ScriptManagerError.remoteScriptNotAllowed)
     }
 
+    // MARK: - Exporting scripts
+
+    func testExportingScriptWithoutPathThrows() {
+        assert(try run(with: ["export"]), throwsError: ExportError.missingPath)
+    }
+
+    func testExportingScriptWithNameThrows() {
```

Comment:
Perhaps give this a bit more descriptive name, like `testExportingMissingScriptThrows()`